packages/fx-core/src/plugins/resource/sql/errors.ts

```diff
@@ -91,7 +91,8 @@ export class ErrorMessage {
 
   public static readonly SqlAccessError = {
     name: "SqlAccessError",
-    message: (sqlName: string, detail = "") => `Failed to access server '${sqlName}'. ${detail}`,
+    message: (sqlName: string, ip: string, detail = "") =>
+      `Failed to access server '${sqlName}', please refer the help to add '${ip}' into the server firewall rule since your public ip may occasionally change. ${detail}`,
```

Comment:
I don't think this needs to change. The '${detail}' contains enough information.

Reply:
The change adds redundant information.

Reply:
It is fixed. Thanks.